src/main/java/io/debezium/connector/yugabytedb/YBClientUtils.java

```diff
@@ -247,6 +248,47 @@ public static OpId getOpIdFromGetTabletListResponse(GetTabletListToPollForCDCRes
     return null;
   }
 
+    /**
```

Comment:
The design of this functions seems convoluted. The ultimate goal is to get CDCStreamInfo. However to get that, a table UUID is required. So two rpc calls are required. Not a perf concern but a convoluted logic concern. Why doest `getDBStreamInfoResponse` return `CDCStreamInfo` as well?